pkg/controller/test_helper.go

```diff
@@ -215,5 +205,6 @@ func BuildConfig() (*rest.Config, error) {
 	if err != nil {
 		return nil, err
 	}
+	config.Timeout = 30 * time.Second
```

Comment:
Possibly move this to a `const` so someone knows what this timeout is configuring.

Reply:
Done in 6b6ec81